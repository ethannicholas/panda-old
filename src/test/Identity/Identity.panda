Console.writeLine(5 == 5)
Console.writeLine("Hello" == "Goodbye")
