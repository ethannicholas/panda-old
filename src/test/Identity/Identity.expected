error: Identity.panda:1:21: '==' cannot operate on 'panda.core.Int8', 'panda.core.Int8'
error: Identity.panda:2:27: '==' cannot operate on 'panda.core.String', 'panda.core.String'
2 errors
